### DISCUSSION TITLE
airflow Task Not Yet started

### DISCUSSION THREAD

**Discussion by LostAsk** [apache/airflow#26621] [General]
### Apache Airflow version

Other Airflow 2 version

### What happened

python version  3.9.9

run example_python_operator, user BashOperator,  official demo   
![image](https://user-images.githubusercontent.com/46678241/191887403-24e2e1ad-9aed-4b0f-b6f9-53df8aa7788f.png)

but run example_external_task_marker_parent
![image](https://user-images.githubusercontent.com/46678241/191887092-c20e9838-2afb-4aea-8c52-112511faff07.png)

### What you think should happen instead

run example_external_task_marker_parent
![image](https://user-images.githubusercontent.com/46678241/191887092-c20e9838-2afb-4aea-8c52-112511faff07.png)

### How to reproduce

Run the official demo

### Operating System

CentOS Linux release 7.4.1708 (Core) 

### Versions of Apache Airflow Providers

Version:2.3.4

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Feel free to fix it, but unless you want to take care about it, more context is needed. When, what, what logs, what is your deployment, what are the logs indicating it. I run the demo on stock 2.3.4 and could not see any problematic behaviour, so you need to make some investigation and see what is special in your environment that causes it.

![image](https://user-images.githubusercontent.com/595491/191919980-4cdadd59-2033-4e41-8cb4-d99dd2d0c5a1.png)

BTW. You likly have differnt version. 2.3.4 example DAG has also python virtualenv example as you can see.
